curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

```diff
@@ -174,7 +174,7 @@ assert.isAbove($('img')[0].alt.length, 0);
   <p>Welcome to the ultimate guide for making mini chocolate chip cookies! These bite-sized treats are perfect for
     satisfying your sweet tooth without overindulging. Follow this simple recipe to create delicious,
     crispy-on-the-outside, chewy-on-the-inside mini chocolate chip cookies that everyone will love.</p>
-  <img src="cookies-image.jpg" alt="a tray of chocolate chip cookies">
+    <img src="https://cdn.freecodecamp.org/curriculum/labs/recipe.jpg" alt="Ingredients for baking: three eggs, a bowl of flour, a glass of milk, and a whisk arranged on a wooden table.">
```

Comment:
```suggestion
  <img src="https://cdn.freecodecamp.org/curriculum/labs/recipe.jpg" alt="Ingredients for baking: three eggs, a bowl of flour, a glass of milk, and a whisk arranged on a wooden table.">
```

if you can restore the indentation to what it was, there was no reason to change it, that would be better, thank you

Reply:
Thank you for the suggestion. I'll restore the indent as you said.